schunk_description/urdf/pw70/pw70.urdf.xacro

```diff
@@ -0,0 +1,180 @@
+<?xml version="1.0"?>
+<robot xmlns:xacro="http://www.ros.org/wiki/xacro">
+
+  <xacro:include filename="$(find schunk_description)/urdf/pw70/pw70.gazebo.xacro" />
+  <xacro:include filename="$(find schunk_description)/urdf/pw70/pw70.transmission.xacro" />
+
+  <xacro:macro name="schunk_pw70" params="parent name *origin is_upsidedown">
+
+    <xacro:unless value="${is_upsidedown}">
+
+      <joint name="${name}_joint" type="fixed" >
+        <xacro:insert_block name="origin" />
+        <parent link="${parent}" />
+        <child link="${name}_link" />
+      </joint>
+
+      <link name="${name}_link">
+        <xacro:default_inertial/>
+	<visual>
+          <origin xyz="0 0 0.07" rpy="0 0 0" />
+          <geometry>
+            <mesh filename="package://schunk_description/meshes/pw70/pw70_square_holder.stl" />
+          </geometry>
+        </visual>        
+<!--        <collision>
```

Comment:
We left it commented out since it is not well tested. We use the same description as for the visualisation and I am not sure that is the right thing. In this way there is a place holder when someone will work with collisions.